sentry-android-core/src/main/java/io/sentry/android/core/ActivityFramesTracker.java

```diff
@@ -68,7 +72,11 @@ public synchronized void setMetrics(
 
     SparseIntArray[] framesRates = null;
     try {
-      framesRates = frameMetricsAggregator.remove(activity);
+      if (activity != null) {
+        framesRates = frameMetricsAggregator.remove(activity);
+      } else {
+        framesRates = frameMetricsAggregator.getMetrics();
```

Comment:
That will return values for all activities, but our transaction is bound to activity x, also, this won't remove/reset the values for that specific activity.